release-notes/3.0/preview/api-diff/preview8/.Net/3.0-preview8.md

```diff
@@ -0,0 +1,10 @@
+# API Difference netcoreapp3.0-preview7 vs netcoreapp3.0-preview8
```

Comment:
We haven't diff'd APIs in non-System namespaces like `Microsoft.AspNetCore.*`. What motivated doing so now?

Let's keep the file/folder structure consistent with the rest of the preview1-7 api-diffs (i.e. drop the `.Net` directory).
`release-notes/3.0/preview/api-diff/preview8/3.0-preview8.md`

Reply:
@ahsonkhan I discussed this with @danroth27 yesterday, and he requested that we start adding API diffs for Asp.Net. He suggested we do it here in the dotnet/core repo, since Asp.Net is moving more towards being in the "dotnet family". The idea is to have one unified place where customers can see the whole of our API diff.